### PR TITLE
Minor editing of framework components for readability

### PR DESCRIPTION
**Pull request by sarah11918**
Some small edits to polish the page

### CODE REVIEW

File: src/pages/en/core-concepts/framework-components.md [hunk 1]
```diff
@@ -86,12 +92,12 @@ There are serveral hydration directives available for UI framework components: `
 
 ## Mixing Frameworks
 
-An **Astro component** can import and render components from multiple frameworks.
+You can import and render components from multiple frameworks in the same Astro component.
 
-⚠️ *Note that components must be imported including their file extensions.*
+>⚠️ *Only **Astro** components (`.astro`)can contain components from multiple frameworks.*
```
Comment (sarah11918) [inline src/pages/en/core-concepts/framework-components.md:97]:
```suggestion
>⚠️ *Only **Astro** components (`.astro`) can contain components from multiple frameworks.*
```

**Comment by Yan-Thomas** [comment]
Loved the changes! I also didn't find any typo.

File: src/pages/en/core-concepts/framework-components.md [hunk 2]
```diff
@@ -56,9 +56,15 @@ import MyReactComponent from '../components/MyReactComponent.jsx';
...
 ```
 
+>💡 _Remember: all imports must live at the **top** of your Astro component script!_
+
+By default, your framework components will render as static HTML. This is useful for templating components that are not interactive and avoids sending any unnecessary JavaScript to the client.
+
 ## Hydrating Interactive Components
 
-A framework component can be hydrated using a `client:*` directive. This is a component attribute to define how your component should be **rendered** and **hydrated**. It describes whether your component should be rendered at build-time, and when your component's JavaScript should be loaded by the browser, client-side.
+A framework component can be made interactive (hydrated) using a `client:*` directive. This is a component attribute to define how your component should be **rendered** and **hydrated**.
```
Comment (delucis) [inline src/pages/en/core-concepts/framework-components.md:65]: Nit? This might help underline `client:*` itself is not a valid client directive.

```suggestion
A framework component can be made interactive (hydrated) using one of the `client:*` directives. This is a component attribute to define how your component should be **rendered** and **hydrated**.
```

Comment (delucis) [reply src/pages/en/core-concepts/framework-components.md:65]: (otherwise this all looks great!)

**Comment by sarah11918** [comment]
Thanks @Yan-Thomas and @delucis for trying to get me to the weekend!  ;)

File: src/pages/en/core-concepts/framework-components.md [hunk 3]
```diff
@@ -86,12 +92,12 @@ There are serveral hydration directives available for UI framework components: `
```
Comment (jcha0713) [inline src/pages/en/core-concepts/framework-components.md:92]: This is not part of the commit but I guess still a typo.
serveral -> several

**Review by jcha0713 — COMMENTED** [review]
Otherwise all look good to me!
